src/server/generate-manifest-json.js

```diff
@@ -0,0 +1,27 @@
+const { writeFileSync } = require('fs')
+const { resolve } = require('path')
+const { colors } = require('../../dist/server/js/styles/common')
+
+const manifestTemplate = `
+  {
+    "version": "1.1",
+    "name": "nearForm Hacker News",
+    "short_name": "nearForm HN",
+    "icons": [
+      {
+        "src": "/images/favicon-1024.png",
+        "sizes": "1024x1024",
+        "type": "image/png"
+      }
+    ],
+    "start_url": "/",
+    "display": "standalone",
+    "orientation": "portrait",
+    "background_color": "${colors.NEARFORM_BRAND_MAIN}",
+    "theme_color": "${colors.NEARFORM_BRAND_MAIN}"
+  }
+`
```

Comment:
I would really prefer if this was the result of `JSON.stringify` instead.

Reply:
It has been refactored to generate at build time but wouldn't that be more costly?